browser/extensions/brave_extension_provider_browsertest.cc

```diff
@@ -65,4 +67,68 @@ IN_PROC_BROWSER_TEST_F(BraveExtensionProviderTest, PDFJSInstalls) {
   ASSERT_TRUE(pdfjs_exists);
 }
 
+// Load an extension page with an ad image, and make sure it is NOT blocked.
+// It would otherwise be blocked though if it wasn't an extension.
+IN_PROC_BROWSER_TEST_F(BraveExtensionProviderTest, AdsNotBlockedByDefaultBlockerInExtension) {
```

Comment:
Nearly the same test is done in the adblock service tests, but with the result of ads blocked 1 and a different call to setExpectations to indicate that the test expects an item to be blocked.